packages/docusaurus-init/templates/classic/docs/gettingStarted.md

```diff
@@ -0,0 +1,71 @@
+---
+id: getting-started 
```

Comment:
I would use the filename `getting-started.md` and omit this frontmatter: simpler